packages/framework/src/Twig/ImageExtension.php

```diff
@@ -244,4 +264,59 @@ protected function getImageHtmlByEntityName(array $attributes, $entityName, $add
             'imageCssClass' => $this->getImageCssClass($entityName, $attributes['type'], $attributes['size']),
         ]);
     }
+
+    /**
+     * @param array $attributes
+     * @return bool
+     */
+    protected function shouldUseLazyLoad(array $attributes): bool
```

Comment:
I am not sure about this method name. What about `isLazyLoaded` or `isLazyLoadEnabled`?

Reply:
renamed to `isLazyLoadEnabled` in 73e9ce93a